src/Plot.tsx

```diff
@@ -34,39 +25,26 @@ function reducer(state: State, action: ReducerActions) {
       state.series = seriesFiltered;
       break;
     }
-    case 'minMax': {
-      const { min, max, axis } = action.value;
-      if (axis === 'x') {
-        state.xMin = min;
-        state.xMax = max;
+    case 'newAxis': {
+      const { id, position, ...values } = action.value;
+      let currentAxis = state.axis[id];
+      if (currentAxis) {
+        validatePosition(currentAxis.position, position, id);
+        state.axis[id] = { ...currentAxis, position, ...values };
       } else {
-        state.yMin = min;
-        state.yMax = max;
+        state.axis[id] = { position, ...values };
       }
       break;
     }
-    case 'padding': {
-      const { min, max, axis } = action.value;
-      if (axis === 'x') {
-        state.paddingLeft = min;
-        state.paddingRight = max;
-      } else {
-        state.paddingBottom = min;
-        state.paddingTop = max;
-      }
-      break;
-    }
-    case 'flip': {
-      const { flip, axis } = action.value;
-      if (axis === 'x') {
-        state.xFlip = flip;
-      } else {
-        state.yFlip = flip;
-      }
+    case 'removeAxis': {
+      const { id } = action.value;
+      const seriesFiltered = state.series.filter((series) => series.id !== id);
```

Comment:
that seems weird to filter the series on an action called "removeAxis" ?

Reply:
Oups, good catch